apps/azure-functions-api/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "azure-functions-api",
+  "version": "0.0.1",
+  "description": "Azure Function REST API Template",
+  "main": "./dist/main.js",
+  "license": "MIT",
+  "scripts": {
+    "clean": "shx rm -rf dist",
+    "clean:generated": "shx rm -rf ./src/generated",
+    "build": "tsc",
+    "build:watch": "yarn build -w",
+    "typecheck": "yarn build --noemit",
+    "extensions:install": "func extensions install",
+    "start": "dotenv -e .env func start --javascript",
+    "start:watch": "yarn start watch",
+    "lint": "yarn lint:check --fix",
+    "lint:check": "eslint . -c .eslintrc.js --ext .ts,.tsx --cache",
+    "lint:api": "sh api-lint.sh",
+    "api:bundle": "bundle-api-spec -i api/internal.yaml.template -o api/internal.yaml -V $npm_package_version",
+    "generate": "turbo run generate:definitions-internal",
```

Comment:
workspaces should not rely on `root workspace` dependencies (`turbo` is not allowed here)

Reply:
What do you mean?

Reply:
Fixed in 75b9571